autokeras/auto_model.py

```diff
@@ -42,16 +44,17 @@ def __init__(self,
                  max_trials=100,
                  directory=None,
                  objective='val_loss',
+                 tuner='greedy',
```

Comment:
This should also optionally accept a tuner instance (like for layer activations, initializers, etc). This makes the API more flexible (can change the tuner without having to fork the codebase)

Reply:
Since the tuner classes are extending the kerastuner's tuner class. The hypermodel is a required arg for the initializer, which cannot be set directly by the user.
I can make the tuner arg a tuner class instead of an instance.